### DISCUSSION TITLE
Is there a mining program suitable for Android phones?

### DISCUSSION THREAD

**Discussion by wllbmoczuse** [Chia-Network/chia-blockchain#1922] [Q&A]
The use of OTG to mount the hard disk array on Android phones seems to meet the mining standards. But is there this mining program now? Raspberry Pi is fine, but old phones are even better.

**Comment by hoffmang9** [comment]
People in the community have gotten farming a plot to work on Android tablets. For now the software needs 64 bit operating systems and plotting is likely to always need 64 bits - but wallet and maybe farming can be ported to 32 bit versions.
